packages/common/dto/placement-preferences.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { IsBoolean, IsOptional } from 'class-validator';
```

Comment:
Are the validations in these final? If so there are some tickets to update in Jira

Reply:
These validations are no final, this is just to have something to work off of.